src/templates/pad.html

```diff
@@ -476,6 +476,7 @@ <h1>Skin Builder</h1>
   <script type="text/javascript">
       // @license magnet:?xt=urn:btih:8e4f440f4c65981c5bf93c76d35135ba5064d8b7&dn=apache-2.0.txt
       var clientVars = {};
+      clientVars.randomVersionString = '<%=settings.randomVersionString%>';
```

Comment:
Won't this be overwritten when the `CLIENT_VARS` message is sent to the client? See: https://github.com/ether/etherpad-lite/blob/f0a6236a0db683fc4ed5eed29e04e7ee11897e1f/src/static/js/pad.js#L297

It would probably be better to set the value in the `CLIENT_VARS` message here: https://github.com/ether/etherpad-lite/blob/f0a6236a0db683fc4ed5eed29e04e7ee11897e1f/src/node/handler/PadMessageHandler.js#L1038

Reply:
Will change to assigning in one line.
It's intended that it will be replaced later. The reason is that `clientVars` is pushed after receiving `CLIENT_READY` from a client, but clients need to request `plugin-definition.json` before they send their `CLIENT_READY` message.